File: Baekjoon_Online_Judge/NO_08393/8393.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 08                 | #
# | Day 02                   | #
# | 8393 합                  | #
# ---------------------------- #


if __name__ == "__main__":
    n = int(r_input())

    print(n * (1 + n) // 2)

File: Baekjoon_Online_Judge/NO_08393/8393.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 08                 | 
 | Day 02                   | 
 | 8393 합                  | 
 ---------------------------- 
*/

int main(){
    int n, total = 0;

    cin >> n;

    for(int i = 1; i <= n; i ++) total += i;

    cout << total;
    
    return 0;
}
